Makefile

```diff
@@ -83,6 +83,7 @@ end-to-end-test: shared
 	sudo docker ps --quiet --all --filter 'ancestor=welder/web-with-coverage' | sudo xargs --no-run-if-empty docker rm -f
 
 build-rpm:
+	sed -i "s,var welderApiHost=.*,var welderApiHost=\"api\";," ./public/js/config.js
```

Comment:
IMHO this is wrong here. `build-rpm` is not exclusively a test thing, one would expect to use `make srpm` and `make rpm` for RPMs that you can actually install on some machine. Thus this sed should move into `Dockerfile` and become an environment variable so that the `end-to-end` and `cockpit` tests can set that, but "production" usage stays unaffected.

Reply:
@henrywang : BTW, in #190 I introduce a change to pass a different port to the cockpit and welder-web containers. Something similar can/should be used here (i. e. configure it at container run time, not statically in the rpm or container image).

Reply:
@martinpitt: I created a new PR #191, which removed some lines of ugly sed codes the above included.
I'll make above change out of this PR to make this PR only for timeout issue in local environment.